### ISSUE TITLE
Explorer or Search should get keyboard focus when last editor is closed

### ISSUE THREAD

**Issue by jacobweber** [Microsoft/vscode#49841]
- VSCode Version: 1.23.1
- OS Version: Mac OS 10.13.4

Steps to Reproduce:

1. Use arrow keys to select a file in Explorer (or Search).
2. Press command-down to open the file in an editor.
3. Press command-w to close the editor. No more editors are open at this point.
4. The Explorer (or Search) doesn't have keyboard focus anymore.

This makes it difficult to navigate files with the keyboard. For example, you may want to use the arrow keys to select a file, open it to take a look, close it, then continue navigating with the arrow keys to select a different file.

Does this issue occur when all extensions are disabled?: Yes
